dbt_project.yml

```diff
@@ -12,15 +12,24 @@ vars:
     integration: "{{ ref( 'stg_klaviyo__integration') }}" 
     person: "{{ ref( 'stg_klaviyo__person') }}" 
     metric: "{{ ref( 'stg_klaviyo__metric') }}" 
-    
+
+    klaviyo__email_attribution_lookback: 120 # hours - 5 days
```

Comment:
can we add a link to the klaviyo article on where these numbers were discussed? I would find that helpful as a user of the package.

Reply:
yeah definitely -- i believe it's in the README but i should maybe add it in other places?